stacker/blueprints/raw.py

```diff
@@ -186,7 +188,22 @@ def rendered(self):
             template_path = get_template_path(self.raw_template_path)
             if template_path:
                 with open(template_path, 'r') as template:
-                    self._rendered = template.read()
+                    if len(os.path.splitext(template_path)) == 2 and (
+                            os.path.splitext(template_path)[1] == '.j2'):
+                        self._rendered = Template(template.read()).render(
+                            context=self.context,
+                            mappings=self.mappings,
+                            name=self.name,
+                            # self.resolved_variables are not available during
+                            # template rendering; config variables are
+                            # provided directly instead
+                            variables=list(
```

Comment:
Hmm - does this mean we can't use lookups effectively here?

Reply:
Yeah, this was too much of caveat to endure. Fixed now.